default.nix

```diff
@@ -142,6 +145,24 @@ rec {
     fi
   '';
 
+  run-integration-tests-zemu = pkgs.writeShellScriptBin "run-integration-tests" ''
```

Comment:
do GH actions use that?

Reply:
yes,
it is used specially for ledger testing in Makefile
```
nix-integration-test-ledger: check-network 
	nix run -f ./default.nix run-integration-tests-zemu -c run-integration-tests "pytest -v -m ledger"		

```